### ISSUE TITLE
Support for Gatsby 3.0

### ISSUE THREAD

**Issue by emarchak** [Debens/gatsby-plugin-scss-typescript#21]
Just wondering when this will support 3.0 upgrade. Right now it's still tied to 2.1.0

**Comment by Debens** [comment]
I'll be honest here, I've dropped a little from the Gatsby ecosystem recently and ended up being taken by other projects.

But I'm happy to look into upgrading as I doubt it will be much of a change. However, I doubt I'll get around to starting this until next week 🤞  I can do this and the removal of node-sass at the same time.

Feel free to create a PR yourself, or an example repo to give me a head start. Apologies for not being faster to react!

**Comment by Debens** [comment]
Apologies for the delay, this wasn't the over the weekend job I wanted it to be, I ran into a few more issues than I anticipated with how Gatsby now supports tree shaking and had to push this back to the holiday weekend.

The good news is that I now have something working on my example repo, I'm going to publish the changes to the alpha channel for now as I'm not 100% confident that I've covered all bases so I would really appreciate it if you find fellows can give it a run 🙏 If I don't hear anything over the next few days/week I'll push the big red deploy button.

```sh
yarn add gatsby-plugin-scss-typescript@alpha
```

Thank you for your patience!
